code-challenges.test.js

```diff
@@ -52,5 +121,31 @@ const numbersToAdd2 = [0, 7, -8, 12]
 const numbersToAdd3 = []
 // Expected output: []
 
+describe("addItUp", () => {
+
+    it("take numbersToAdd1 and pass the values and return array of the accumulating sum", () => {
+
+        expect(addItUp()).toEqual([2, 4, 45, 9])
+
+    })
+})
+
 
 // b) Create the function that makes the test pass.
+
+//---- I created a function called addItUp
+//---- I created numbersToAddSort function to sort than add the sum to the current indx value 
+
+const addItUp = () => { 
+
+const numbersToAddSort = numbersToAdd1.sort((a, b) => a + b)
+ 
+const rackItUp = numbersToAddSort.map((sum => value => sum += value)(0))
+      
+      return rackItUp
+}
+console.log(addItUp())
+
+//---- numbersToAdd1 console.logged: [ 2, 6, 51, 60 ]
+//---- numbersToAdd2 console.logged: [ -8, -8, -1, 11 ]
+//---- numbersToAdd3 console.logged: []
```

Comment:
you are missing the parameter  being passing into the function on line 139.  This should then be used on line 141 instead of hardcoding numbersToAdd1.  Your logic is on point and this function will work on all cases if you just take those added steps to make it dynamic.